vmm/src/vm_config.rs

```diff
@@ -645,4 +796,94 @@ pub struct VmConfig {
     // valid, and will be closed when the holding VmConfig instance is destroyed.
     #[serde(skip)]
     pub preserved_fds: Option<Vec<i32>>,
+    #[serde(default)]
+    pub landlock_enable: bool,
+    pub landlock_config: Option<Vec<LandlockConfig>>,
```

Comment:
I'm a bit confused if this config is here (in `VmConfig`) then it it VM configuration not VMM configuration (like seccomp) and then will be automatically exposed via the HTTP API. Can you clarify whether this is VM configuration or VMM configuration?

Reply:
I initially started with `vmm-config` like seccomp, but later moved it to `VmConfig`. I moved to VmConfig to allow for `LandlockConfig` to serialized and de-seriliazed along with rest of the VM configs during live migration.

Reply:
It appears this is more than changing the group. If we don't want those to be part of VmConfig, then the application of the rules needs to be rewritten.